tools/make/diplomat-coverage/src/allowlist.rs

```diff
@@ -295,9 +295,14 @@ lazy_static::lazy_static! {
         "icu_provider_adapters::fork::ForkByErrorProvider",
         "icu_provider_adapters::fork::predicates::ForkByErrorPredicate",
 
+        // Borrowed types
+        "icu::collator::CollatorBorrowed",
+        "icu::collator::CollatorBorrowed::try_new",
```

Comment:
nit: add these as hidden `rustlink`s on the relevant APIs

Reply:
Added.

Reply:
and remove here

Reply:
Removed.